static/src/javascripts/projects/commercial/modules/header-bidding/prebid/bid-config.js

```diff
@@ -59,7 +59,8 @@ import {
 } from '../utils';
 import { getAppNexusDirectBidParams } from './appnexus';
 
-const PAGE_TARGETING: {} = buildAppNexusTargetingObject(getPageTargeting());
+// The below line is needed for page skins to show
+getPageTargeting();
```

Comment:
Is this because page skins are part of the targeting?

Reply:
It shouldn't matter because the targeting is already been set from the prepare-googletag module here:  
https://github.com/guardian/frontend/blob/2f9c1c9cf1e059c5d0c52ccc074e54a19d4322d1/static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js#L66

Reply:
¯\_(ツ)_/¯